builtin-functions/kphp-light/functions.txt

```diff
@@ -215,11 +195,6 @@ function warning($message ::: string) ::: void;
 /** @kphp-no-return */
 function critical_error($message ::: string) ::: void;
 
-function debug_print_string($str ::: string) ::: void;
-
-function byte_to_int($str ::: string) ::: ?int;
```

Comment:
I added this functions for proptest. Maybe, we really doesn't need them anymore

Reply:
I think that we need to rewrite proptest with new component API, so we really don't need these functions now.